### PR TITLE
DOC-2561: Remove trailing zero from dates in release-notes.py

### PR DESCRIPTION
**Pull request by nickvigilante**
Release note: None

### CODE REVIEW

File: scripts/release-notes.py [hunk 1]
```diff
@@ -833,7 +833,7 @@ def analyze_standalone_commit(commit):
     print("summary: Additions and changes in CockroachDB version", current_version, "since version", previous_version)
     print("---")
     print()
-    print("## " + time.strftime("%B %d, %Y"))
+    print("## " + time.strftime("%B %e, %Y"))
```
[the remark this answers is not in the document]
Comment (nickvigilante) [reply scripts/release-notes.py:836]: Lemme circle back to my team and ask them. I got it from http://strftime.net/.

**Review by rafiss — APPROVED** [review]
lgtm!

**Comment by nickvigilante** [comment]
TFTR!

**Comment by nickvigilante** [comment]
bors r=rafiss

**Pull request merged**